docs/scale-tests/styles.css

```
@@ -0,0 +1,337 @@
+  cursor: pointer;
+  border: 1px solid var(--border);
+  color: var(--muted);
+  background: transparent;
+  transition: all 0.1s;
+  user-select: none;
+}
+.filter-pill:hover:not(.active) { border-color: var(--text); color: var(--text); }
+.filter-pill.pass.active { border-color: var(--pass); color: var(--pass); background: rgba(63,185,80,0.08); }
+.filter-pill.fail.active { border-color: var(--fail); color: var(--fail); background: rgba(248,81,73,0.08); }
+.filter-pill.skip.active { border-color: var(--skip); color: var(--skip); background: rgba(210,153,34,0.08); }
+.dot { width: 7px; height: 7px; border-radius: 50%; background: currentColor; }
```

Stylelint (17.4.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)